docs/src/components/Layout/Header.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import { useState } from 'react';
```

Comment:
I want to say there is a reason we use `import * as React from 'react';` as opposed to `import {} from 'react';` but I don't remember why.

Reply:
We prefer the `import * as React from 'react';` for consistency across the codebase and minimizing import statement code changes. See: 
https://epicreact.dev/importing-react-through-the-ages/

Reply:
Thanks for sharing! It's fun to know all the histories on how to import react.

Looks like both are fine, but I'll convert my change back for consistency :D